### ISSUE TITLE
sfx changer

### ISSUE THREAD

**Issue by ThePotatoGuy** [Monika-After-Story/MonikaModDev#9592]
probably be a dropdown (can we do that?) in settings

things to consider:
* need json to map things?
* could we just base it on filenames or something? 
* do we need to organize sfx better?

**Comment by ThePotatoGuy** [comment]
lol there u get something

**Comment by Booplicate** [comment]
*Sees new assignment: UI in renpy*
*Retires*

**Comment by ThePotatoGuy** [comment]
lol ok this prob a good starter for someone new - team is kind of smol these days

**Comment by dreamscached** [comment]
Could you provide more clear details? What exactly is needed and how?

**Comment by Booplicate** [comment]
json is to create sfx packs, akin to how spritepacks work.
There will be UI in the setting for selecting a pack.

**Comment by Booplicate** [comment]
I think this should instead be done together with support for custom UI themes, so SFX can be part of the theme.

**Comment by dreamscached** [comment]
Yea that sounds reasonable. Maybe we could upgrade spritepacks to resource packs actually?

**Comment by Booplicate** [comment]
What do you mean by resource packs? Ship spritepacks together with UI themes?

**Comment by dreamscached** [comment]
Not really, rather make uniform sort of package that would include various resources, which can be just dropped somewhere around DDLC folder to be picked up by MAS at startup.

That would ensure users don't mess up, as well as let users create their own resource packages which would include new sprites and other assets.

Adding submods to this list would sound reasonable to me, but the new submod framework is already in works so I guess that wasn't the best idea I've had.

**Comment by Booplicate** [comment]
Well, this should be in a separate issue for discussion.
Generally I'd like spritepacks to be self-contained akin to the new submods, but I don't think this is possible while keeping things compatible with the old style, @ThePotatoGuy probably knows better, though.
There's also #9619 which should improve the structure of accessories.

**Comment by ThePotatoGuy** [comment]
sfx is like assets, spritepacks are like content. combining sfx with UI elements make sense, but not with spritepacks, and only maybe with submods.

if we imagine this as a steam game:
* spritepacks are cosmetic DLC
* sfx and UI are workshop asset mods
* submods are workshop script mods
